### ISSUE TITLE
Engineering Reference and possibly code for window luminance calculation is inconsistent with technical literature

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#4776]
Helpdesk ticket 9883
This is not a bug report, but rather a technical comment regarding the window luminance calculation.
The EP documentation (Eng. Ref) states that for a bar window (no shades), the window luminance for direct sunlight is calculated as the product of the sun luminance times the cosine of the incidence angle on the window surface times the window transmittance:

Lw = Lsun * Trans_win * COS(B). See equation in page 187 of the Eng. Ref.

This is no correct. The COS(B) is double counted in the calculation of the illuminance from a window. See equation (161) and the equation before it, page 186.

The correct window luminance should read:

Lw = Lsun * Trans_win . So if there is no window (Trans = 1), the luminance of the opening is equal to that of the source.

This is consistent with the expression in the publication:

Winkelmann, F.C. and S. Selkowitz. 1985. Daylighting Simulation in the DOE-2 Building Energy Analysis Program. Energy and Buildings 8, 271-286.

If this is not a typo, the window luminance calculation module in EP should be revised.

Note: I did some simulations, but the results I got for the window luminance (and Glare) were not consistent under sunny sky conditions from one hour to another (despite the sky is clear, except the COS(B) changes). This might suggest that the window luminance is wrongly implemented in EP? See figure below for a south facing window:

![image001](https://cloud.githubusercontent.com/assets/4991075/6537210/0a9a8462-c41a-11e4-8399-7436109ef163.png)

**Comment by lgu1234** [comment]
@mjwitte I would like to take this CR. Do you have an electronic copy of the reference?

Winkelmann, F.C. and S. Selkowitz. 1985. Daylighting Simulation in the DOE-2 Building Energy Analysis Program. Energy and Buildings 8, 271-286.

I searched Science Direct and found the abstract only due to early publication.

**Comment by mjwitte** [comment]
@lgu1234 No, I do not.  Perhaps you can ask the user who posted the helpdesk ticket.

**Comment by lgu1234** [comment]
@mjwitte I found it in the LBL website.

**Comment by Myoldmopar** [comment]
@lgu1234, you mentioned above you wanted to take this issue.  Are you able to address it?

**Comment by lgu1234** [comment]
@Myoldmopar I will take a look to see whether I can address it or not.

**Comment by lgu1234** [comment]
I looked at #4776 again. Here are my findings.

1 Eng. Reference

I looked at the reference cited in Eng. Ref. User’s observation is correct. The equation in Eng. Ref. is not correct, compared to the Winkelmann's paper. It should be without COS(B):

Lw = Lsun \* Trans_win 

2 E+ outputs

A figure provided by the user presents direct beam and south luminance. The luminance results are strange with solar beam radiation > 0 during day time. I tried to reproduce the figure using Chicago weather, since the user did not mention which weather file was used and the input file shows that Chicago design days are used. Unfortunately, I was unable to reproduce. Then I sent a request to the user in HelpDesk on 9/22/15 and did not get any feedback.  

![windowsolarandilluminace](https://cloud.githubusercontent.com/assets/5970656/13010458/e02055a4-d16f-11e5-9770-8f3c7055043b.jpg)

The above figure was generated on the same day using Chicago weather. I don’t see any strange results. Although the direct normal is close to zero, the diffuse radiation is still above zero. Therefore, the illuminance results are OK with me.

Here is my opinion to handle this CR.
1. Correct the equation in Eng. Ref
2. Unless we can reproduce the user’s results, I assume OK with current calculations. Since I was unable to get feedback from the user, it is possible that the user may make a mistake to generate the figure.

**Comment by mjwitte** [comment]
@lgu1234 Just for completeness, can you point to a line in the source code that shows this equation?

**Comment by lgu1234** [comment]
@mjwitte Although WLUMSK, WLUMSU are used to represent L_w in Eng. Ref, these two variables with break points are not stopped in debug. But statements do not include COSB.. 
Line 7162 in DayltgInterReflectedIllum of DaylightingManager: 
                        WLUMSK( IHR, 1, ISky ) += ZSK( ISky ) \* TVISBR / Pi;
Line 7171in DayltgInterReflectedIllum of DaylightingManager:
                            WLUMSU( IHR, 1 ) += ZSU \* TVISBR / Pi;

However, a section starting from Line 7218 uses variables FLFWSK and FLCWSK

```
                    for ( ISky = 1; ISky <= 4; ++ISky ) {
                        //IF (PH < 0.0d0) THEN
                        //Fixed by FCW, Nov. 2003:
                        if ( PH > 0.0 ) {
                            FLFWSK( 1, ISky ) += ZSK( ISky ) * TVISBR;
                            if ( ISky == 1 ) FLFWSU( 1 ) += ZSU * TVISBR;
                        } else {
                            FLCWSK( 1, ISky ) += ZSK( ISky ) * TVISBR;
                            if ( ISky == 1 ) FLCWSU( 1 ) += ZSU * TVISBR;
                        }

                    }
```

The calculation has no COSB also. These two variables are eventually used in calculation of output variable: SurfaceWindow( SurfLoop ).IllumFromWinAtRefPt1Rep to report "Daylighting Window Reference Point 1 Illuminance [lux]".

**Comment by mjwitte** [comment]
Closed via #5487
